test/unit/replayTest.js

```diff
@@ -21,10 +26,15 @@ test('replay', t => {
   t.equal(replayLog.getLog().length, 1);
   t.equal(replayLog.getLog()[0].sprites.length, 0);
 
-  p5Inst.createSprite();
+  p5Inst.createDanceSprite();
   replayLog.logFrame({p5: p5Inst});
   t.equal(replayLog.getLog().length, 2);
   t.equal(replayLog.getLog()[1].sprites.length, 1);
 
+  p5Inst.createEffectsSprite();
```

Comment:
nit: I'd love to either see a comment explaining what this is testing, or see this test broken up into a couple different tests with descriptive names